src/hera/validators.py

```diff
@@ -3,7 +3,7 @@
 from typing import Optional
 
 
-def validate_name(name: str, max_length: Optional[int] = None) -> str:
+def validate_name(name: str, max_length: Optional[int] = None, generate_name: bool = False) -> str:
```

Comment:
Small: any chance `generate_name: bool = False` can be part of the docstring?

Reply:
Added docstring for both generate_name and max_length